src/PotentialCalculation/PotentialCalculationSetup/PotentialCalculationSetupCartesian3D.jl

```diff
@@ -188,6 +188,8 @@ function PotentialCalculationSetup(det::SolidStateDetector{T}, grid::CartesianGr
                         ig = find_closest_gridpoint(pt, point_types.grid)
                         if is_undepleted_point_type(point_types.data[ig...]) 
                             ϵ[ix, iy, iz] *= scaling_factor_for_permittivity_in_undepleted_region(det.semiconductor) * (1 - imp_scale[ig...])
+                        elseif is_fixed_point_type(point_types.data[ig...])
+                            ϵ[ix, iz, iz] *= scaling_factor_for_permittivity_in_undepleted_region(det.semiconductor)

```

Comment:
Still a typo: it is `iy`, not `iz`

Reply:
It looks like these cases are not covered by our tests?
Do you have a MWE where this is important that we can add to the tests?

Reply:
Ah, I thought it is tested in the test set "Depletion Handling: PN Junction", but I just realized that only the electric potential is calculated there.